test/articledownloadlinkslist.html

```diff
@@ -43,51 +43,52 @@ <h3 class="article-download-links-list__heading">Downloads<span class="visuallyh
 
         >Figures</a></li>
       </ul>
-      <h3 class="article-download-links-list__heading">Download citations<span class="visuallyhidden"> (links to download the citations from this article in formats compatible with various reference manager tools)</span></h3>
+      <h3 class="article-download-links-list__heading">Open citations<span class="visuallyhidden"> (links to open the citations from this article in various online reference manager services)</span></h3>
```

Comment:
Should we rename this now we have other component elements  for the titles i.e. Cite this article